sys/include/net/gnrc/tcp.h

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2015 Simon Brummer
+ * Copyright (C) 2017 Simon Brummer
```

Comment:
I know that payment is not the only criteria but with payment comes usually a contract there you give up your copyright and I can't remember that I signed something like that during my time at the HAW. I just prefer to keep my rights if I can choose to do so.

I didn't want to start a copyright discussion either. My somebody here knows for sure if its is okay to keep my name up there ;)